### PR TITLE
Release ODFE 1.11.0

### PR DESCRIPTION
**Pull request by penghuo**
*Description of changes:*
ODFE 1.11.0 Release, Elasitcsearch 7.9.1, Kibana 7.9.1
Release notes: https://github.com/penghuo/sql/blob/od-1-11-0-release/release-notes/opendistro-for-elasticsearch-sql.release-notes-1.11.0.0.md

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by abbashus — APPROVED** [review]
Looks good.

File: release-notes/opendistro-for-elasticsearch-sql.release-notes-1.11.0.0.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+* Bug fix, order by doesn't work when group by field has alias ([#766](https://github.com/opendistro-for-elasticsearch/sql/pull/766))
+* Bug fix, using matcher to compare the Json result ([#762](https://github.com/opendistro-for-elasticsearch/sql/pull/762))
+
+
+### Infrastructure
+* Enforce 100% branch coverage for sql and es module ([#774](https://github.com/opendistro-for-elasticsearch/sql/pull/774))
+
+### Workbench
+* Restyle workbench SQL/PPL UI ([#781](https://github.com/opendistro-for-elasticsearch/sql/pull/781))
+* UI Separate SQL and PPL pages ([#761](https://github.com/opendistro-for-elasticsearch/sql/pull/761))
```
Comment (joshuali925) [inline release-notes/opendistro-for-elasticsearch-sql.release-notes-1.11.0.0.md:28]: Can we move these two into enhancements? ODFE has a category standard on release notes now

**Review by joshuali925 — COMMENTED** [review]
Thanks

https://github.com/penghuo/sql/blob/od-1-11-0-release/sql-jdbc/build.gradle#L35
this should also be 1.11.0.0

File: release-notes/opendistro-for-elasticsearch-sql.release-notes-1.11.0.0.md (hunk 1 above)
Comment (penghuo) [reply release-notes/opendistro-for-elasticsearch-sql.release-notes-1.11.0.0.md]: Done.

**Review by joshuali925 — APPROVED** [review]

**Pull request merged**